### DISCUSSION TITLE
Option to replace default indicator

### DISCUSSION THREAD

**Discussion by fabulousfastboy** [mclemente/about-face#69] [Ideas]
I would love to be able to swap the default facing indicator out for a custom one, including rotation and offset options. Something like the turn indicator module, which comes with a few options by default but also allows using custom images. Would be very useful for GURPS heheh.

**Comment by mclemente** [comment]
About Face's indicator isn't an image, it's a drawing: it's set of points with a line being drawn over them and then being filled. So setting your own indicator would require you to play around with coordinates. It is doable, but isn't that simple.
I had a request for a thinner indicator earlier and it'll be coming out as a setting on the next update of the module, so I'll see how it goes for making more presets.

**Reply by mclemente** [reply]
Also, for your rotation/offset options, there is the `Snap Indicator to Closest Tile` setting, which was made by one of the GURPS system developers to solve issues with the module.
